scripts/camera_and_video.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+###############################################################################
+# Copyright 2018 The Apollo Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###############################################################################
+
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd "${DIR}/.."
+
+source "${DIR}/apollo_base.sh"
+
+function start() {
+    LOG="${APOLLO_ROOT_DIR}/data/log/camera_and_video.out"
+    CMD="cyber_launch start /apollo/modules/drivers/camera/launch/camera_and_video.launch"
+    NUM_PROCESSES=`expr "$(pgrep -c -f "modules/drivers/camera/dag/camera.dag")" + "$(pgrep -c -f "modules/drivers/video/dag/video.dag")"`
+    if [ "${NUM_PROCESSES}" -eq 0 ]; then
+       eval "nohup ${CMD} </dev/null >${LOG} 2>&1 &"
+    fi
+}
+
+function stop() {
+    eval "nohup cyber_launch stop /apollo/modules/drivers/camera/launch/camera_and_video.launch < /dev/null 2>&1 &"
+}
+
+# run command_name module_name
+function run() {
+    case $1 in
+        start)
+            start
+            ;;
+        stop)
+            stop
+            ;;
```

Comment:
What about adding a `restart` option here?

Reply:
Good suggestion! We may prefer though to create a separate PR to add `restart` options for all related scrips together.